common/artifact_spawns/wc_artifact_spawns.txt

```diff
@@ -485,4 +485,253 @@ aluneth = {
+	}
+    artifacts = {
+		felomelorn = { value = 1 }
+    }
+}
+hammer_of_the_twilight = {
+	max_amount = 1
+    spawn_date = 583.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+		modifier = {
+			factor = 0
+			NOT = { year = 585 }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			is_playable = yes
+			
+			culture = twilights_hammer
+		}
+	}
+    artifacts = {
+		hammer_of_the_twilight = { value = 1 }
+    }
+}
+hammer_of_the_lightbringer = {
+	max_amount = 1
+    spawn_date = 583.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 60036						# Uther Lightbringer
+		}
+	}
+    artifacts = {
+		hammer_of_the_lightbringer = { value = 1 }
+    }
+}
+flute_of_accuracy = {
+	max_amount = 5
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			OR = {
+				character = 42086						# Alleria Windrunner
+				character = 42002						# Anasterian Sunstrider
+				character = 42089						# Idorial Windrunner
+			}
+		}
+	}
+    artifacts = {
+		flute_of_accuracy = { value = 1 }
+    }
+}
+lionhorn_of_stormwind = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 1008						# Anduin Lothar
+		}
+			additive_modifier = {
+			value = 100
+			character = 8							# Varian Wrynn
+		}
+	}
+    artifacts = {
+		lionhorn_of_stormwind = { value = 1 }
+    }
+}
+warsong_battle_drums = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			has_landed_title = d_warsong			# Grommash Helscream
+		}
+	}
+    artifacts = {
+		warsong_battle_drums = { value = 1 }
+    }
+}
+horn_of_cenarius = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 410002						# Cenarius
+		}
+	}
+    artifacts = {
+		horn_of_cenarius = { value = 1 }
+    }
+}
+spellbook_of_drekthar = {
+	max_amount = 1
+    spawn_date = -1.1.1
+	
+	spawn_chance = {
+		value = 100
+		modifier = {
+			factor = 0
+			NOT = { has_dlc = "Mystics" }
+		}
+	}
+    
+    weight = {
+		value = 0
+		additive_modifier = {
+			value = 100
+			character = 10100						# Drek'thar
+		}
+	}
+    artifacts = {
+		spellbook_of_drekthar = { value = 1 }
+    }
+}
```

Comment:
Have it in 583

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/67306546-d6420700-f4ff-11e9-961c-7f33e65818aa.png)

</details>

Reply:
get's it in 2. bookmark